src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -302,7 +304,7 @@ static List<ActiveRunner> InitializeRunners(RunDescriptor runDescriptor, IList<E
 
                 if (endpointName.Length > 77)
                 {
-                    throw new Exception(string.Format("Endpoint name '{0}' is larger than 77 characters and will cause issues with MSMQ queue names. Please rename your test class or endpoint.",endpointName));
+                    throw new Exception(string.Format("Endpoint name '{0}' is larger than 77 characters and will cause issues with MSMQ queue names. Please rename your test class or endpoint.", endpointName));
```

Comment:
That was there before. It is just a reformating. We could argue that check belongs to the ConfigureMsmqTransport class.